### DISCUSSION TITLE
Cross Validation Evaluate Usage with upper limit prediction instead of y_pred

### DISCUSSION THREAD

**Discussion by HishamSalem** [alan-turing-institute/sktime#2817] [Q&A]
does the evaluate function offer the ability to change the y_pred in the 'return_data' function to the upper/lower interval when using expanding window cross validation?

Or do we just have to subclass the function?

**Comment by fkiraly** [comment]
I think currently it can´t do that. We have recently been working on upgrading metrics and cross-validation splitters to hierarchical, evaluation would be next. Contributions are appreciated.
